subsys/logging/log_cmds.c

```diff
@@ -363,6 +363,30 @@ static int cmd_log_backends_list(const struct shell *shell,
 	return 0;
 }
 
+static int cmd_log_strdup_utilization(const struct shell *shell,
+				      size_t argc, char **argv)
+{
+	u32_t buf_cnt = log_get_strdup_pool_utilization();
+	u32_t buf_size = log_get_strdup_longest_string();
+
+	shell_print(shell,
+		"Maximal utilization of the buffer pool: %d of %d available.",
```

Comment:
The text might be a bit confusing. "available" seems to refer to the second number, but the text might also be interpreted as how many buffers out of a total are available (i.e. referring to the first number). So I'd consider rewording this, or perhaps just dropping "available" and using "%d / %d"

Reply:
good point. fixed.